### DISCUSSION TITLE
x-intersect

### DISCUSSION THREAD

**Discussion by radohingis** [alpinejs/alpine#1726] [1. Help]
I would like to sincerely ask, what do I do to use x-intersect safely without throwing errors using npm package? I'm not going to copy whole project. Is there a way to implement this directive to project via npm? like 'npm i @alpinejs/intersect' after I installed 'alpinejs' simply? bc it didn't work :\ Thank you for help. 

(btw. I worked around my own implementation, but it throws me an error that MutationObserver is not defined, and I have no idea in this stage, using vite-ssg, where it should be :DD)

**Comment by joshhanley** [comment]
@radohingis the intersect plugin hasn't officially been released yet, so if you want to use it, you will need to pull it down from this repo manually and added it in. 

Hope this helps!
